src/popup/pages/FungibleTokens/TokenContainer.vue

```diff
@@ -123,16 +123,17 @@ export default defineComponent({
   setup(props, { root }) {
     const { getSdk } = useSdk();
 
+    const tokenType = root.$route.meta?.isCoinDetails ? 'coin' : 'token';
```

Comment:
Also I suggest to avoid using magic strings.

Reply:
I think @peronczyk suggested to use constants defined in `routeNames`.